lib/iris/_cube_coord_common.py

```diff
@@ -34,7 +34,7 @@ class LimitedAttributeDict(dict):
                        'coordinates', 'grid_mapping', 'climatology',
                        'cell_methods', 'formula_terms', 'compress',
                        'missing_value', 'add_offset', 'scale_factor',
-                       'valid_max', 'valid_min', 'valid_range', '_FillValue')
```

Comment:
It would make sense to ensure that Iris isn't writing out these attributes in an manner that was inconsistent with CF.